test/compress/harmony.js

```diff
@@ -217,10 +217,9 @@ import_statement: {
 
 import_all_statement: {
     input: {
-        import * from 'lel';
```

Comment:
This is just invalid code.

Reply:
Oops!

Reply:
This should be a parse error. Nobody's noticed since 2015 so we should be fine though :)